sw/airborne/firmwares/rotorcraft/guidance/guidance_h_ref.h

```diff
@@ -54,24 +54,19 @@ extern float gh_max_speed;
 #define GH_FREQ_FRAC 9
```

Comment:
interesting, it seems that the integration frequency of 512 is implicitly encoded here...
this is not used anymore (you have added a proper `dt`) and should be removed I think